frontend/packages/ceph-storage-plugin/src/components/block-pool/body.scss

```diff
@@ -0,0 +1,3 @@
+.ceph-block-pool-body__input {
+    margin: var(--pf-global--spacer--lg) 0 !important;
+}
```

Comment:
multiple components are body component, so let maintain it now CSS in separate file